### PR TITLE
Publish Checkout Templates for Velo

### PR DESCRIPTION
**Pull request by wix-docs-feds**

### CODE REVIEW

File: wix-ecom-backend/wix-ecom-backend/CheckoutTemplates/guides/Introduction.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+&nbsp;
+> **Note:** This module is
+> [universal](/api-overview/api-versions#universal-modules).
+> Functions in this module can run on both the backend and frontend,
+> unless specified otherwise.
+
+The Checkout Templates API allows you to create and manage pre-populated checkouts to share with customers. When customers then create checkouts from these templates, the new checkouts already include this information.
+
+With the Checkout Templates API, you can:
+Run a sale on a specific product for a limited time
```
Comment (roysha-wix) [inline wix-ecom-backend/wix-ecom-backend/CheckoutTemplates/guides/Introduction.md:18]: this line and the one below render as the same sentence.
I believe you may have forgotten bullet points?

File: wix-ecom-backend/wix-ecom-backend/CheckoutTemplates/guides/Introduction.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Limit a sale offer to a select number of checkouts
+Customize the checkout experience
+
+## Before you begin
+
+It’s important to note that the checkout template functionality is not currently available in the Wix Dashboard. If you create checkout templates with the API, you will also manage and update your checkout templates through the API. For example, when you offer a sale through a checkout template you need the API to change the template’s `status` to `INACTIVE` in order to turn off the sale when you want it to end.
+
+To use the Checkout Templates API, import { checkoutTemplates } from the ‘wix-ecom-backend’ module:
```
Comment (roysha-wix) [inline wix-ecom-backend/wix-ecom-backend/CheckoutTemplates/guides/Introduction.md:26]: add backticks to the relevant text

**Review by roysha-wix — APPROVED** [review]
approved to alleviate merging.
left a couple of comments here, and couple more in slack

**Comment by AriGoldWix** [comment]
Will publish in autodocs-wix-ecom-backend branch. Closing this PR.

**Pull request closed without merging**